src/hidet/runtime/cuda/cuda.cpp

```diff
@@ -79,3 +94,32 @@ DLL void hidet_cuda_set_device(int device) {
     lazy_load_cuda_runtime();
     CHECK_CUDA(cudaSetDevice(device));
 }
+
+DLL void* hidet_cuda_malloc(size_t size) {
+    lazy_load_cuda_runtime();
+    void* devPtr = malloc(sizeof(void*));
```

Comment:
```suggestion
    void* devPtr;
```
We do not need to allocate a memory region.

Reply:
Maybe I'm missing something, but wouldn't doing this result in devPtr being created on the stack, and then we'd be returning a local stack variable?

Reply:
When we call 
```c++
cudaMallocAsync(&devPtr, ...)
```
we are passing the address of the pointer to the cuda api function, which will update the pointer value. The pointer is a stack variable and will be valid during we calling the cuda api function. We returns the "value" of the pointer instead of the "address" of the pointer to the callee of `hidet_cuda_malloc`, this is fine.

Reply:
Got it. Thanks!